content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -17,23 +17,19 @@
 
         <div class="row">
 
-     <div class="col-md-8">
-          <h3>Objectives</h3>
-                <ul>
-                    <li>Scale an app using kubectl.</li>
-                </ul>
-            </div>
-
             <div class="col-md-8">
+
+              <h3>Objectives</h3>
+              <ul>
+                  <li>Scale an app using kubectl.</li>
+              </ul>
+
        <h3>Scaling an application</h3>
 
             <p>Previously we created a <a href="/docs/concepts/workloads/controllers/deployment/"> Deployment</a>, and then exposed it publicly via a <a href="/docs/concepts/services-networking/service/">Service</a>. The Deployment created only one Pod for running our application.  When traffic increases, we will need to scale the application to keep up with user demand.</p>
             <p>If you haven't worked through the earlier sections, start from <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">Using minikube to create a cluster</a>.</p>
 
             <p><em>Scaling</em> is accomplished by changing the number of replicas in a Deployment</p>
```

Comment:
Very minor _nit_ in the existing content.
```suggestion
            <p><em>Scaling</em> is accomplished by changing the number of replicas in a Deployment.</p>
```